File: Basic Programming/Hackerrank 30 days of code c++ solution/Day_26.cpp
/*                    Name : Arif Khan
                      Judge: HACKERRANK
                      University: Primeasia University
                      problem: Day 26: Nested Logic
                      Difficulty: Easy
                      Problem Link: https://www.hackerrank.com/challenges/30-nested-logic/problem
*/

#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
   int d1,m1,y1,d2,m2,y2;
    cin>>d1>>m1>>y1>>d2>>m2>>y2;
    if(y1>y2) return cout<<10000, 0;
    else if(y1 == y2){ 
        if(m1>m2) return cout<<(m1-m2)*500,  0;
        else if(m1 == m2 and d1 > d2) return cout<<(d1-d2)*15, 0;
    }
    cout<<0<<endl;
    return 0;
}

File: Basic Programming/Hackerrank 30 days of code python3 solution/Day_26.py
#                      Name : Arif Khan
#                      Judge: HACKERRANK
#                      University: Primeasia University
#                      problem: Day 26: Nested Logic
#                      Difficulty: Easy
#                      Problem Link: https://www.hackerrank.com/challenges/30-nested-logic/problem
#

d1, m1, y1 = map(int, input().split())
d2, m2, y2 = map(int, input().split())

ans = 0
if y1 > y2:
    ans = 10000
elif y1 == y2:
    if m1 > m2:
        ans = (m1-m2)*500
    elif m1==m2 and d1 > d2:
        ans = (d1-d2)*15
print(ans)